packages/components/src/components/modal/modal.tsx

```diff
@@ -38,6 +38,7 @@ type TModalElement = {
     title?: string | React.ReactNode;
     toggleModal?: (e?: React.MouseEvent<HTMLElement>) => void;
     width?: string;
+    shouldCloseOnEscape?: boolean;
```

Comment:
`shouldCloseOnEscape: boolean;`
Is because that by default `shouldCloseOnEscape` is equals to `true` by default